galette/templates/default/pages/configuration_entitleds.html.twig

```diff
@@ -12,7 +12,21 @@
 {% block javascripts %}
     <script type="text/javascript">
         $(function() {
-            {% include 'elements/js/removal.js.twig' %}
+            {% set type = path_for("doAddEntitled", {"class": url_class})|split("/")|slice(1, 2)[0] %}
```

Comment:
Hum... I wonder if there is no more clean solution; this seems hard to understand.

Reply:
yes : `{% set type = url_class %}` :rofl: